.gitignore

```diff
@@ -61,6 +61,5 @@ bazel-testlogs
 .secrets
 opm
 faq
-jobs/*
```

Comment:
This was done simply to force the required CI checks to run. We don't have a jobs directory, so should be safe.